build/IPDK_Container/scripts/run_cleanup.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+#Copyright (C) 2021 Intel Corporation
+#SPDX-License-Identifier: Apache-2.0
+
+ROOT_DIR=$1
+KEEP_SOURCE_CODE=$2
+
+FLAG_YES="YES"
+FLAG_NO="NO"
+
+if [ "${KEEP_SOURCE_CODE,,}" = "${FLAG_NO,,}" ] ; then
+    echo "Copy p4ovs_env_setup.sh script from P4-OVS"
+    cp "${ROOT_DIR}/P4-OVS/p4ovs_env_setup.sh" "${ROOT_DIR}/scripts/"
+
```

Comment:
Why not always copy the scripts to the scripts dir? In that way there is no need for different paths for starting and using just like @vsureshkumarp mentioned above.